plugins/search/providers/elastic_search/lib/model/search/ESearchQueryAttributes.php

```diff
@@ -113,6 +118,22 @@ public function setObjectId($objectId)
 		$this->objectId = $objectId;
 	}
 
+	/**
+	 * @return KalturaNullableBoolean
```

Comment:
cannot us this type here as you are in core and this type comes from the higher level of the API